### PR TITLE
Very Simple install script

### PR DESCRIPTION
**Pull request by Tehiro**
It's not perfect - i know
You are free! - Do what you want - it works :)

### CODE REVIEW

File: media/style.css [hunk 1]
```diff
@@ -22,6 +22,7 @@ table, caption, tbody, tfoot, thead, tr, th, td {
 	outline: 0;
 }
 
+install
```
Comment (Tehiro) [inline media/style.css]: My bad, delete this please.

**Comment by Tehiro** [comment]
So that you do not use?

**Comment by ddinchev** [comment]
@Tehiro what is the purpose of your changes? This is just a school project, do you use it and want to contribute?

If you do, keep in mind that I changed mysql php module to mysqli.

**Comment by Tehiro** [comment]
If this is school project - sorry.

**Pull request closed without merging**